src/constants.rs

```diff
@@ -93,8 +93,8 @@ pub const BASEPOINT_ORDER: Scalar = Scalar {
 
 use crate::ristretto::RistrettoBasepointTable;
 /// The Ristretto basepoint, as a `RistrettoBasepointTable` for scalar multiplication.
-pub const RISTRETTO_BASEPOINT_TABLE: RistrettoBasepointTable =
-    RistrettoBasepointTable(ED25519_BASEPOINT_TABLE);
+pub static RISTRETTO_BASEPOINT_TABLE: &'static RistrettoBasepointTable =
```

Comment:
I think that this has to stay `static` because it's a new type and doesn't point to the same table (notice that this means that the table might appear twice in the binary once for ed25519 and once for ristretto).

This can probably be bypassed(but sadly stay as static) with some `#[repr(transparent)]` and unsafe (https://play.rust-lang.org/?gist=c73adb96a26f2fbc90e67e250f1750de)

Reply:
Okay, 78d79bb uses "const crimes" to cast the static reference to `EdwardsBasepointTable` to a now `#[repr(transparent)]` `RistrettoBasepointTable`, which avoids making a copy of the table.